README.md

```diff
@@ -6,12 +6,12 @@ A list of resources connected to the [SpaceTraders API](https://spacetraders.io/
 
 ## Contents
 
-* [Awesome SpaceTraders](#awesome-spacetraders)
-  * [Contents](#contents)
-  * [Client](#client)
-  * [SDKs](#sdks)
-  * [Assets](#assets)
-  * [Tutorials](#tutorials)
+- [Awesome SpaceTraders](#awesome-spacetraders)
+  - [Contents](#contents)
+  - [Client](#client)
+  - [SDKs](#sdks)
+  - [Assets](#assets)
+  - [Tutorials](#tutorials)
```

Comment:
Might be your editor which changed the markdown list symbol here, please change back to use `*` instead of `-`.